### PR TITLE
Changes for #77, #80, #81

### PR DESCRIPTION
**Pull request by AnneThKu**
- Added "Training" to support page - https://github.com/subnero1/subnero1.github.io/issues/81
- Added scroll to the respective modem section - https://github.com/subnero1/subnero1.github.io/issues/80
- Added link for custom solutions in products page - https://github.com/subnero1/subnero1.github.io/issues/77

### CODE REVIEW

File: solutions/index.md [hunk 1]
```diff
@@ -25,7 +27,7 @@ Our modems are also available in several editions. The _silver edition_ modems a
 
 Our underwater modems can not only be used for communication, but also for underwater navigation. They can be used for acoustic ranging, and to form a GPS-like infrastructure underwater, aiding underwater vehicles in self-localization as well as communication. The nodes can be used as beacons for position estimation and can transparently relay information. This allows vehicles to navigate accurately as well as communicate over long distances to other vehicles, underwater installations and/or surface nodes. These nodes may be easily dropped into water in an area of operation. Once in water, the nodes self-organize into an Underwater Positioning Network (UPN). This network can then be accessed by AUVs and Remotely Operated Vehicles (ROVs) to estimate their location, navigate and communicate effectively.
 
-<div class='one spacing'></div>
+<div class='one spacing' id="spacing"></div>
```
Comment (notthetup) [inline solutions/index.md:30]: Any reason for this id?

Comment (AnneThKu) [reply solutions/index.md:30]: This was removed in the following commit. But I missed to push the modem.md file which I have pushed now.

**Review by notthetup — APPROVED** [review]

**Comment by manuignatius** [comment]
1. We don't seem to be using a full stop after any email addresses (in general, all throughout the website). This might be something we used to use, but find out whether this is the correct usage before merging

2. Custom solutions, the image says "Custom Configuration". While I understand, the word configuration is used to match with other products above, it gives the impression that there is only 1 custom configuration, which is not true. I would propose removing the entire block and keep it simple by adding a line that goes something like, "Details of our customized solutions are available here.". What we are doing are actually solutions, not products.

3. The addition of anchor is not working correctly, either the anchor used is not correct or its by purpose. Would like to know more.

**Comment by AnneThKu** [comment]
1. Full stop added after email Id.
2. Figure removed and Custom solutions added as below.
image

3. If the anchor is linked to the heading, it would go beneath the top-bar, hence anchor is linked to the tags which would bring the required section into view.

**Comment by AnneThKu** [comment]
Travis build might fail because of some of the Unetstack links, which I will note and correct with #87

**Review by manuignatius — APPROVED** [review]

**Pull request merged**